src/__test__/components/data-exploration/heatmap/__snapshots__/HeatmapPlot.test.jsx.snap

```diff
@@ -1,7 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
-
-exports[`HeatmapPlot dispatches loadCellSets action when no cell sets are in the store 1`] = `
-Object {
-  "type": "cellSets/loading",
-}
```

Comment:
it doesn't seem like you are using this

Reply:
I didn't need the snapshots for this, so it was deleted I guess, why?

Reply:
it looked like you added it .. or I got confused

Reply:
my point was that since this file needs to be removed, because it is not used

Reply:
oh, I think it was removed, I looked in file changes and it said file deleted